lib/router.dart

```diff
@@ -298,11 +299,14 @@ Route<dynamic> createRoute(RouteSettings settings) {
 
     case Routes.addressBook:
       return MaterialPageRoute<void>(
-          builder: (_) => getIt.get<ContactListPage>(param1: true));
+          builder: (_) =>
+              getIt.get<ContactListPage>(param1: true, param2: null));
```

Comment:
**minor**
you can ignore passing it at all, it would be still parsed as null too

Reply:
fixed it